### DISCUSSION TITLE
Is there way to create multiple supplier ?

### DISCUSSION THREAD

**Discussion by OlehObodin** [saleor/saleor#9298] [Q&A]
Hi, is this platform fit marketplace cases? Or can I easy adapt it?

**Comment by karolkielecki** [comment]
Hi @OlehObodin, we do not support such functionality. We have considered marketplace functionality in the past, but it is not something we plan to implement anytime soon. We do have an integration with Jetti.io -- you might check it out as a possible fit. https://jetti.io/integrations/store/saleor
